src/compose.py

```diff
@@ -9,8 +9,13 @@ def start(settings):
 
 
 def stop(settings):
-    command = "down" if settings["persist_data"] else "down --volumes"
-    run(command, settings)
+    run("stop", settings)
```

Comment:
The other behaviour difference we'll see here with `stop` instead of `down` is that the containers won't be removed.  Does that end up being ok with restarts?

Reply:
It does seem to. But maybe we should in fact remove them.